plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ObjectWriteFunction.java

```diff
@@ -28,6 +29,11 @@ void set(PreparedStatement statement, int index, Object value)
             throws SQLException;
 
     static <T> ObjectWriteFunction of(Class<T> javaType, ObjectWriteFunctionImplementation<T> implementation)
+    {
+        return of(javaType, Types.JAVA_OBJECT, implementation);
```

Comment:
I don't know what JAVA_OBJECT means, but it's a wrong default here for sure.

`ObjectWriteFunction` is an interface used to write things that have "Object" representation within Trino type system. For example, a `timestamp(9)`.

Reply:
Fixed.